tests/dummy/app/templates/signup-example.hbs

```diff
@@ -1,20 +1,20 @@
 <h1>Signup example</h1>
 
-<form {{action "signup" on="submit"}}>
+<form class="signup-example-form" {{action "signup" on="submit"}}>
```

Comment:
Not sure if I follow. The helpers will be the same, with the exception that some of them, like `visit` should probably raise an error if used in integration